src/main/java/org/myrobotlab/service/Polly.java

```diff
@@ -356,7 +356,8 @@ public static void main(String[] args) {
 
   @Override
   public boolean isReady() {
-    return polly != null ? true : false;
+    setReady(polly != null ? true : false);

```

Comment:
@kwatters said : the isReady() method should only return this.ready ... the logic for calling this.ready = true should be implemented in the code where the polly service is created and started.. not here

Reply:
Inside the constructor is not enough, we need to check for valid api keys, after service is started. 
invalid api keys > polly==null